src/index.ts

```diff
@@ -560,27 +577,36 @@ exports.isPrecompiled = function (address) {
  * @param {String} str
  * @return {String}
  */
-exports.addHexPrefix = function (str) {
+export const addHexPrefix = function(str: string): string {
   if (typeof str !== 'string') {
     return str
   }
 
-  return exports.isHexPrefixed(str) ? str : '0x' + str
+  return ethjsUtil.isHexPrefixed(str) ? str : '0x' + str
 }
 
 /**
  * Validate ECDSA signature
  * @method isValidSignature
- * @param {Buffer} v
+ * @param {Number} v
```

Comment:
Is this a breaking change introduced here or just fixing the docs?

Reply:
Fixing the docs. `v` seems to be number in all other functions (and tests).